src/components/DataViewFilterDropdownButtons/DataViewFilterDropdownButtons.tsx

```diff
@@ -53,8 +52,7 @@ function DataViewFilterDropdownButtons(props: DataViewFilterDropdownButtonsProps
 				<Button label={t("mosaic:common.clear")} color="black" variant="text" onClick={props.onClear}/>
 			</ButtonRow>
 			<StyledButtonRow>

```

Comment:
I think StyledButtonRow might not be needed anymore because we no longer have 2 buttons, we could update it to just use ButtonRow.